test/controllers/confirmation.controller.spec.unit.ts

```diff
@@ -0,0 +1,76 @@
+import * as request from "supertest";
+import app from "../../src/app";
+import * as pageURLs from "../../src/model/page.urls";
+import {COOKIE_NAME} from "../../src/properties";
+import {loadSession} from "../../src/services/redis.service";
+import {getPromiseToFileSessionValue, updatePromiseToFileSessionValue} from "../../src/services/session.service";
+import {getDummyCompanyProfile, loadCompanyAuthenticatedSession, loadMockSession} from "../mock.utils";
+
+jest.mock("../../src/session/store/redis.store", () => import("../mocks/redis.store.mock.factory"));
+jest.mock("../../src/services/redis.service");
+jest.mock("../../src/client/apiclient");
+jest.mock("../../src/services/session.service");
+
+const COMPANY_NUMBER: string = "00006400";
+const COMPANY_NAME: string = "THE GIRLS DAY SCHOOL TRUST";
+const EMAIL: string = "test@demo.ch.gov.uk";
+const PAGE_TITLE: string = "Confirmation page";
+const URL: string = pageURLs.PROMISE_TO_FILE + "/company/" + COMPANY_NUMBER + "/confirmation";
+const ERROR_PAGE: string = "Sorry, there is a problem with the service";
+
+describe("confirmation screen stating that the company is no longer required", () => {
+
+  const mockCacheService = loadSession as jest.Mock;
+  const mockPTFSession =  getPromiseToFileSessionValue as jest.Mock;
+ 
```

Comment:
lint is failing due to trailing whitespace. Can you run 'make githooks' to add lint to the commit hook.

Reply:
Removed whitespace